tests/FakeItEasy.Specs/CallMatchingSpecs.cs

```diff
@@ -732,23 +732,18 @@ public static void UnusedArgumentMatcherDescriptionNotUsed(IHaveNoGenericParamet
         public static IEnumerable<object[]> Fakes()
         {
             yield return new object[] { A.Fake<object>(), "Faked " + typeof(object) };
-            yield return new object[] { A.Fake<List<int>>(), "Faked " + typeof(List<int>) };
-            yield return new object[] { A.Fake<IList<int>>(), "Faked " + typeof(IList<int>) };

```

Comment:
Why remove `List<int>`?

Reply:
Because from xUnit's perspective, both test cases are the same (both have an empty collection and the "[]" string), and it only runs one of them.

Reply:
Darn you to heck, xUnit! Thanks.